### PR TITLE
Revert "WINC-755: [upi-vsphere] Support Windows Server 2022"

### PR DESCRIPTION
**Pull request by saifshaikh48**
Reverts openshift/release#28430

SSH problems with the golden image, only in UPI environment (platform none).

### CODE REVIEW

**Comment by sebsoto** [comment]
/hold
ensure this fixes issue before merge

**Comment by sebsoto** [comment]
pj rehearse failed due to subsciption-manager issue
but the timing of the image change pr is too coincidental for it to not be the issue IMO
/approve

**Review by jrvaldes — APPROVED** [review]
/lgtm

**Pull request merged**
